dex/models/_projects/paraswap/paraswap_trades.sql

```diff
@@ -19,39 +19,72 @@ ref('paraswap_avalanche_c_trades')
 ,ref('paraswap_base_trades')
 ] %}
 
+WITH basic_trades AS (
```

Comment:
Hi @jeff-dude,
should I use `basic_trades` or simply `trades` here for the CTE naming?
As I may add similar CTE across the spells, so I think we may have an unified naming standard.